var/spack/repos/builtin/packages/spiral/package.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Spiral(CMakePackage):
+    """SPIRAL is a program generation system for linear transforms and other
+    mathematical functions that produces very high performance code for a wide
+    spectrum of hardware platforms."""
+
+    homepage = "https://spiral.net"
+    url      = "https://github.com/spiral-software/spiral-software/archive/8.1.2.tar.gz"
+
+    maintainers = ['spiralgen']
+
+    version('8.1.2', sha256='506f1dbf923aa1c9f19f05444fa947085715eef37c9d2494d133fcaaa1dd50bc')
+
+    extendable = True
+
+    # No dependencies.
+
+    def build(self, spec, prefix):
+        print("starting build phase, working dir = " + self.build_directory)
+        print("Now run 'make all'")
+        with working_dir(self.build_directory):
+            make('all')
+            make('install/local')
```

Comment:
Shouldn't this be part of the `install` phase below?

Reply:
Because of the way spiral is structured, No.  Spiral has C code which is compiled and built (out-of-source-tree), but also many files and packages of interpreted GAP code.  In order to run tests we need to do make "install/local".

Reply:
Does this write to the install prefix?

Reply:
No.  "install/local" only operates on the temporary staging tree.